### ISSUE TITLE
Campaign Landing Page audit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17367]
## Description
Complete a full accessibility audit of the Campaign Landing page product, including:
- https://www.va.gov/initiatives/sign-in-securely-with-logingov/
- https://www.va.gov/initiatives/vote/

## Acceptance Criteria

- [x] Accessibility audit has been completed
- [x] Recommended updates and improvements have been documented
- [x] Any high priority/critical updates are brought to the attention of PM/DM/PO
- [x] Design review ticket is created

**Comment by laflannery** [comment]
## Suggested updates/improvements for CLPs:

**Overall**
- Other than “This page is for”, the small gray text is superfluous, it’s most likely getting skipped by SRs and sometimes it’s redundant (FAQs), can we remove these entirely?
- What you can do, Spotlights, Stories:
   - Consider unlinking headings and adding separate CTAs to align with [upcoming Content guidelines](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2146) (This world probably require CMS work)

**Hero Banner**
- Primary CTA:
  - Currently using a non-component Reverse Action link and will need to be updated when ready - [planned for mid-April](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2500)

**Why this Matters**
- This page is for:
  - Remove unnecessary list role on `<ul>`
  - Can we consider adding the bullets here? Unstyled lists are not always read properly so it really should only be used when absolutely necessary
- Social links
  - Remove unnecessary list role on `<ul>`
  - Each link has an icon, will need to be dealt with in `<va-icon>` migration
  - Links currently open in a new tab because they have the special class - remove the class but also this needs to be dealt with in vets-website (not 100% sure on ownership here)
- Secondary CTA - Will need to be updated to component when available

**What you can do**
- Update the images to have a null alt

**Video**
- Can we move the video description above the video?
  - To me this seems like a better user flow - read what the video is about then you can more easily decide if you want to watch it
  - It is also more consistent with the rest of the sections on the page (Heading, intro text, then stuff)
- CTA:
  - This is currently using `<va-link>` component with the secondary action link class - that combination doesn’t work.
  - It should be using the action style (as the others do) so needs to be taken out of the component
  - It will also need to be changed to the new action component when available

**Spotlight**
- Optional CTA: This isn’t showing as a CTA/Action link like all the others, it’s just tacked onto the end of the into text ([Example](http://preview-staging.vfs.va.gov/preview?nodeId=19951))
  - I recommend reviewing this with design and if this should be more consistent with the other sections and CTAs

**Stories**
- Change the images to null alt, these are decorative images
- Link to other stories - This isn’t showing on the Front end, it also isn't working very well in Drupal (I can’t select other nodes)

**Downloadable resources**
- These should use [Card component](https://design.va.gov/components/card#default---gray)
- Download links should use [link component](https://design.va.gov/components/link/#download)
  - Remove the aria-label that is currently being used here
- CTA: Secondary CTA that will need to be converted when ready ([planned for mid-April](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2500))

**Comment by laflannery** [comment]
@aklausmeier would you want to review/go over the above notes and see how and/or what we might want to ticket for potential next steps? Particularly because I do have some small design/display questions in my comments?

**Comment by aklausmeier** [comment]
@laflannery yes

> would you want to review/go over the above notes
